metrics-clojure-ring/project.clj

```diff
@@ -1,8 +1,8 @@
-(defproject metrics-clojure-ring "2.10.0-SNAPSHOT"
+(defproject malesch/metrics-clojure-ring "2.9.1"
   :description "Various things gluing together metrics-clojure and ring."
   :url "https://github.com/sjl/metrics-clojure"
   :license {:name "MIT"}
-  :dependencies [[cheshire "5.7.0"]
-                 [metrics-clojure "2.10.0-SNAPSHOT"]]
+  :dependencies [[cheshire "5.7.1"]
```

Comment:
Hi Michael

Sorry for the late response! I was away the last three weeks and couldn't react on the change request. Because I needed my modifications I pushed them onto Clojars a few days later with my group name. I didn't realise that the additional commit would be picked up by the PR (refers to the branch and not the SHA what actually makes sense).  I think the best is to make a branch for the change request and create a new PR. I hope this is fine.